### PR TITLE
Change Maven groupId to 'org.hibernate.reactive'

### PR DESCRIPTION
**Pull request by Sanne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Hibernate or JPA.
 
 Add the following dependency to your project:
 
-	org.hibernate.rx:hibernate-rx-core:1.0.0-SNAPSHOT
+	org.hibernate.reactive:hibernate-rx-core:1.0.0-SNAPSHOT
```
Comment (aguibert) [inline README.md:117]: should the artifact IDs be changed to match also? e.g. `hibernate-reactive-core`

**Comment by aguibert** [comment]
@Sanne should java package names also be changed from `rx --> reactive`?

**Comment by DavideD** [comment]
@aguibert We are going to change the packages later because at the moment we are working on several PR and it would cause conflicts

**Comment by Sanne** [comment]
@aguibert that's all valid suggestions but I was keeping this one change at a minimum, so to not interfere with large patches that Gavin and Davide are actively working on.

**Comment by Sanne** [comment]
BTW just to clarify: I wanted to accelerate on the groupId being set already so that we could start setting up the release group on bintray.

File: README.md (hunk 1 above)
Comment (gavinking) [reply README.md:117]: My 2c is that if the group ID is exactly identical to the artifact id, then we have a redundancy and what the hell are artifact ids even for?

**Pull request merged**